pyproject.toml

```diff
@@ -16,6 +16,10 @@ python-dateutil = "^2.9.0.post0"
 setuptools = "*"
 uritemplate = "^4.1.1"
 urllib3 = "^2.2.2"
+requests = "^2.32.3"
+typeguard = "^4.3.0"
+xmlasdict = "^0.2.2"
+pandas = "^2.2.2"
```

Comment:
pandas is quite a heavy dependency - where does this come from?
we might need it eventually, but I don't see how subyt requires this?

Reply:
at first glance only for testing --> so declare the dependency appropriately ?